spec/lib/appsignal/cli/diagnose_spec.rb

```diff
@@ -17,6 +17,7 @@
 
       out_stream.string.should include('Gem version')
       out_stream.string.should include('Agent version')
+      out_stream.string.should include('Environment')
```

Comment:
That is something I have another branch for. I wanted to commit all tests in one go, but that was a bit more tricky than I thought. There's no test for the mkmf.log output for example now.